src/components/models/notificationCategory.js

```diff
@@ -1,12 +0,0 @@
-const notificationCategory = {
-    all: "All",
-    new: "New",
-    analysis: "Analysis",
-    data: "Data",
-    tool_request: "Tool Request",
-    apps: "Apps",
-    permanent_id_request: "Permanent ID Request",
-    team: "Team",
-    community: "Community",
-};
-export default notificationCategory;
```

Comment:
I thought we will are using file names starting with uppercase only for Components 🤔 . Regardless standardizing this will be good.

Reply:
I don't think there's any standard or convention really out there, but for some reason my mind seems to think stuff in the models directory seem better suited as PascalCase. I guess because they're supposed to represent some aspect of a data object model, and object = PascalCase to me?  

Our `index` files are also components but they're not PascalCase so 😄 🤷‍♀️ (not saying they should be PascalCase, just an interesting exception)

Reply:
Agreed. Having models starting with uppercase seems reasonable to me. WRT `index` files, everyone seems to use lowercase for that. I guess that convention is beyond `react` comes from the `js` community but 🤷‍♂️

Reply:
I guess I was also thinking JSON model = object = PascalCase.

I think the `index.js` naming convention goes back to traditional webpage directories, where a URL like `my.site.example.com/some/thing/` would retrieve the document at `my.site.example.com/some/thing/index.html` by default, so an `import SomeThing from "some/thing";` will look for the file `some/thing/index.js` by default. And URLs were traditionally case-insensitive, so I'm used to seeing all lowercase HTML docs and I can't recall ever seeing an `Index.html` file. The `index.js` naming convention is probably also following the same tradition.

So if everyone agrees, I like standardizing on components and models using PascalCase file names.
And things like `util` files, which are just a collection of functions, seem better suited as all lowercase file names.

But I also have no objection to standardizing on lowercase model file names either, and reserving PascalCase strictly for components, if anyone has a strong opinion on that.

Reply:
Lets settle for PascalCase for models.